File: java/103 ExecutorService.java
package code;
import java.util.concurrent.ExecutorService;
import java.util.concurrent.Executors;
public class Test {
	static class PrintNumbers implements Runnable {
        @Override
        public void run() {
            for (int i = 1; i <= 10; i++) {
                System.out.printf("Number is %d%n", i);
            }
        }
    }
    public static void main(String[] args) throws InterruptedException {
    	ExecutorService service = Executors.newSingleThreadExecutor();
            PrintNumbers task = new PrintNumbers();
            service.submit(task);
    }
}

File: python/103 ExecutorService.py
from concurrent.futures import ThreadPoolExecutor

class PrintNumbers:
    def __call__(self):
        for i in range(1, 11):
            print(f"Number is {i}")

executor = ThreadPoolExecutor(max_workers=1)
task = PrintNumbers()
executor.submit(task)
executor.shutdown(wait=True)
